br/pkg/pdutil/pd.go

```diff
@@ -329,6 +329,16 @@ func parseVersion(versionBytes []byte) *semver.Version {
 	return version
 }
 
+// TODO: always read latest PD nodes from PD client
+func (p *PdController) getAllPDAddrs() []string {
+	ret := make([]string, 0, len(p.addrs)+1)
+	if p.pdClient != nil {
+		ret = append(ret, p.pdClient.GetLeaderAddr())
```

Comment:
`PdController` is shared by BR, @3pointer PTAL

Reply:
LGTM